<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeraView</class>
 <widget class="QMainWindow" name="TeraView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Spectrum Analysis</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="MatplotlibWidget" name="spectrumCanvas" native="true"/>
              </item>
             </layout>
            </widget>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QTreeWidget" name="fileTree">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <column>
               <property name="text">
                <string>Plot</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Time Domain Info</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Frequency Domain Info</string>
               </property>
              </column>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Image</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Optical Constants</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1247</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuPlots">
    <property name="title">
     <string>Plots</string>
    </property>
    <widget class="QMenu" name="menuSpectrum_Analysis">
     <property name="title">
      <string>Spectrum Analysis</string>
     </property>
     <addaction name="actionPlot_uncertainty_intervals"/>
     <addaction name="actionPlot_SNR"/>
     <addaction name="actionPlot_Dynamic_Range"/>
     <addaction name="actionNoise_Floor_calculated"/>
     <addaction name="actionNoise_Floor_from_fft"/>
    </widget>
    <addaction name="menuSpectrum_Analysis"/>
   </widget>
   <widget class="QMenu" name="menuData_Operations">
    <property name="title">
     <string>Data Operations</string>
    </property>
    <widget class="QMenu" name="menuTime_Domain">
     <property name="title">
      <string>Time Domain</string>
     </property>
     <addaction name="actionZero_Padding"/>
     <addaction name="actionWindowing"/>
     <addaction name="actionScale_Data"/>
    </widget>
    <addaction name="menuTime_Domain"/>
    <addaction name="actionFrequency_Domain"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlots"/>
   <addaction name="menuData_Operations"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="mainStatus">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionPlot_uncertainty_intervals">
   <property name="text">
    <string>plot uncertainty intervals</string>
   </property>
  </action>
  <action name="actionPlot_SNR">
   <property name="text">
    <string>plot SNR</string>
   </property>
  </action>
  <action name="actionPlot_Dynamic_Range">
   <property name="text">
    <string>plot Dynamic Range</string>
   </property>
  </action>
  <action name="actionNoise_Floor_calculated">
   <property name="text">
    <string>Noise Floor calculated</string>
   </property>
  </action>
  <action name="actionNoise_Floor_from_fft">
   <property name="text">
    <string>Noise Floor from fft</string>
   </property>
  </action>
  <action name="actionFrequency_Domain">
   <property name="text">
    <string>Frequency Domain</string>
   </property>
  </action>
  <action name="actionZero_Padding">
   <property name="text">
    <string>Zero Padding</string>
   </property>
  </action>
  <action name="actionWindowing">
   <property name="text">
    <string>Windowing</string>
   </property>
  </action>
  <action name="actionScale_Data">
   <property name="text">
    <string>Modify Data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
